.gitignore

```diff
@@ -104,3 +104,6 @@ ENV/
 # All kinds of vim stuff
 **/*~
 *.sw[a-z]
+
+# visual studio code configuration
+.vscode
```

Comment:
This should be actually a different commit, not talking about a different PR ... ;-)
But yes this is good to have